### ISSUE TITLE
Accessibility issue: Elements must meet minimum color contrast ratio thresholds on /notebook-ci-testing/notebooks/ACS/acs_sbc_dark_analysis/acs_sbc_dark_analysis.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#199]
## What
An accessibility scan flagged an element on https://spacetelescope.github.io/notebook-ci-testing/notebooks/ACS/acs_sbc_dark_analysis/acs_sbc_dark_analysis.html because elements must meet minimum color contrast ratio thresholds. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=playwright.

The following element needs attention:

- `<a class="divlink" href="http://docs.astropy.org/en/stable/table/construct_table.html#empty-array-of-a-known-size">
    constructing a table of known size.
</a>` (selector: `.alert-block > .divlink`)

To fix this, ensure the contrast between foreground and background colors meets wcag 2 aa minimum contrast ratio thresholds.

Specifically:

Fix any of the following:
-   Element has insufficient color contrast of 4.1 (foreground color: #176de8, background color: #cff4fc, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
